dowhy/causal_refuters/graph_refuter.py

```diff
@@ -1,69 +1,133 @@
 import logging
 from dowhy.causal_refuter import CausalRefuter, CausalRefutation
-from dowhy.utils.cit import partial_corr
+from dowhy.utils.cit import partial_corr, conditional_MI
+import numpy as np
 
 class GraphRefuter(CausalRefuter):
     """
     Class for performing refutations on graph and storing the results
     """
-    def __init__(self, data, method_name):
+    def __init__(self, data,method_name_discrete = "conditional_mutual_information",method_name_continuous = "partial_correlation"):
         """
         Initialize data for graph refutation
 
         :param data:input dataset
-        :param method_name: name of method for testing conditional independence
-
+        :param method_name_discrete: name of method for testing conditional independence in discrete data
+        :param method_name_continuous: name of method for testing conditional independece in continuous data
         :returns : instance of GraphRefutation class
         """
-        self._refutation_passed = False  
+        self._refutation_passed = "Undecided"  
```

Comment:
I know I had recommended "undecided", but looking at the code, it is simply used for initialization. So then I think that `None` is better. It also does not mix boolean and string for the same variable